docs/ref/forms/api.txt

```diff
@@ -932,6 +932,50 @@ and using the template above, would render something like:
 
     <label for="myFIELD">...</label><input id="myFIELD" type="text" name="my_field" />
 
+.. _customizing-bound-field:
+
+Customizing ``BoundField``
+--------------------------
+
+.. versionadded:: 1.9
+
+If you are in a situation where using a custom subclass of
+:class:`~django.forms.Field` isn't enough, consider customizing
```

Comment:
maybe something like "If you need to access some additional information about a form field in a template and using a subclass of :class:`~django.forms.Field` isn't sufficient, consider ..."